src/Core/src/Handlers/WebView/WebViewHandler.Windows.cs

```diff
@@ -162,6 +168,11 @@ async void SendNavigated(string url, WebNavigationEvent evnt, WebNavigationResul
 			CurrentNavigationEvent = WebNavigationEvent.NewPage;
 		}
 
+		void SendProcessFailed(CoreWebView2ProcessFailedEventArgs args)
+		{
+			VirtualView?.ProcessTerminated(new WebProcessTerminated(PlatformView, args));
```

Comment:
Sender might need to be `CoreWebView2` and not the framework element.

Reply:
I think the sender should be whoever actually sent _this_ event, which in this case is the MAUI control or handler. It happens to be wrapping some other event.

Reply:
The platform events are meant to pass the sender of the actual native event. These values must match the values passed in by the platform event handler.

Reply:
Sounds a bit odd to me but if that's the pattern let's stick to that.

Reply:
this is the platform args in the event args. So the main sender is the view, then n the platform args it is the sender from the platform. Nested event args!

Reply:
🐢
🐢
🐢
🐢
🐢
🐢
🐢
🐢
🐢

Reply:
Ok, applied changes. Now the Sender is CoreWebView2.